### ISSUE TITLE
Choose the default timeline based on the preferred group

### ISSUE THREAD

**Issue by tio-trom** [friendica/friendica#10629]
This is not a big deal but I thought since Friendica supports RSS feeds and you may end up like me and have lots of such feeds, to be able to choose your default timeline when you click the "Network" icon. So I have Friends and Feeds for instance, and I'd like the default to be Friends. idk is a small think, not super important, but could be useful. Especially since Friendica remembers what you chose last in terms of type: "Latest activity", "Latest posts", and so forth. So maybe it would work well if it would remember the group too.

Anyways, that's my little suggestion.

**Comment by tobiasd** [comment]
Have a look at the _startpage_ addon. You can set any page within Friendica as your personal startpage, including the network stream with a special contact group like your "Friends".

**Comment by tio-trom** [comment]
Thanks. But that only changes it after login. If I click the Network icon it will redirect to all of the posts from all contacts. You know what I mean?

**Comment by tobiasd** [comment]
So, you'd like to redirect the target of your "Network Stream" to that specific group per default.

**Comment by tio-trom** [comment]
Yup, or to remember last selected group the same way it remembers if I select "Latest activity", "Latest posts", and so forth.
